<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prodi</class>
 <widget class="QMainWindow" name="Prodi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prodi</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableData">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>270</y>
      <width>661</width>
      <height>221</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Kode Prodi</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nama Prodi</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="inptKode">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="inptNama">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kode Prodi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nama Prodi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>303</x>
      <y>200</y>
      <width>113</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdate">
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>200</y>
      <width>113</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>523</x>
      <y>200</y>
      <width>113</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>200</y>
      <width>113</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
